pallets/liquidation-pools/src/lib.rs

```diff
@@ -427,10 +456,15 @@ pub struct Sales {
 
 impl<T: Config> Pallet<T> {
 	fn _offchain_worker(_now: T::BlockNumber) -> Result<(), OffchainErr> {
-		// Check if we are a potential validator
+		// Check if we are a potential validator and balancing is enabled.
 		if !sp_io::offchain::is_validator() {
 			return Err(OffchainErr::NotValidator);
 		}
+
+		if !Self::pool_balancing_state_storage() {
+			return Err(OffchainErr::PoolsBalancingIsOff);
+		}
```

Comment:
```suggestion
		ensure!(Self::pool_balancing_state_storage(), OffchainErr::PoolsBalancingIsOff);
```
this suggestion can be applied for check in lines 460-462.